README.md

```diff
@@ -770,6 +772,20 @@ Migrated 3 configuration file(s) from previous versions (0 not migrated)
 Your configuration file(s) are now into [~/git/misc/PostgreSQL/pgenv/config]
 ```
 
+
+The `use` command accepts a specific version that will be used to compute the
+configuration file name related to such version, and the environment variable
+`PGENV_CONFIGURATION_FILE` will be printed out.
+It is required that variable to be exported or used within the shell session
+for subsequent `pgenv` invocations to use such configuration file. For example:
+
+```
+export $( pgenv config use 15.4 )
```

Comment:
I think `use` isn't the right term to use here. Maybe `path` or `show-path`, and just print the path, too, rather than the environment declaration. Then one could use it like this:

``` sh
export PGENV_CONFIGURATION_FILE="$(pgenv config show-path default)"
```

Reply:
substituted with `path`.